src/main/java/org/jabref/model/search/SearchFieldConstants.java

```diff
@@ -20,7 +20,7 @@ public enum SearchFieldConstants {
     PAGE_NUMBER("pageNumber"),
     MODIFIED("modified");
 
-    public static final Analyzer LINKED_FILES_ANALYZER = new LinkedFilesAnalyzer();
+    public static final Analyzer LINKED_FILES_ANALYZER = new EnglishAnalyzer();
```

Comment:
We don't need lowecase filter any more?

Is this covered by test cases? (e.g., a PDF with capital letters and a search searching for that word)?

Reply:
The [EnglishAnalyzer](https://github.com/apache/lucene/blob/68cc8734ca28a9db800e4192a636d3b490cfd41a/lucene/analysis/common/src/java/org/apache/lucene/analysis/en/EnglishAnalyzer.java#L101-L110) also uses a lowercase filter. It's better to use this for the PDFs to get more relevant matches because it applies the [PorterStemFilter](https://github.com/apache/lucene/blob/68cc8734ca28a9db800e4192a636d3b490cfd41a/lucene/analysis/common/src/java/org/apache/lucene/analysis/en/PorterStemFilter.java#L26).